processing/process_ihmt.py

```diff
@@ -219,7 +221,11 @@ def process_run(name_source, layout, run_data, out_dir, temp_dir):
     ihmtw_img.to_filename(ihmtw_file)
 
     # Calculate ihMTR
+    # Is there an advantage to keeping ihmt_files_t1space as a list?
+    # It's less intuitive than using variables
```

Comment:
I wanted to loop over the images, but there are more intuitive ways like you said. Maybe a dictionary so it's still easier to organize the image types in a single variable I can index?